controller/array_action/array_mediator_xiv.py

```diff
@@ -131,18 +138,21 @@ def _generate_snapshot_response(self, cli_snapshot):
                         cli_snapshot.wwn,
                         cli_snapshot.name,
                         self.endpoint,
-                        volume_id=cli_snapshot.copy_master_wwn,
+                        volume_id=self._get_copy_source_object_wwn(cli_snapshot),
                         is_ready=True,
                         array_type=self.array_type)
 
-    def get_volume(self, volume_name, pool=None):
-        logger.debug("Get volume : {}".format(volume_name))
+    def _get_cli_object_by_name(self, volume_name):
```

Comment:
Why is it called CLI? Is it related to Command Line Interface?

Reply:
yes, it's the data you see when using the xcli